synth.py

```diff
@@ -29,4 +29,9 @@
       bazel_target=f'//google/{service}/{version}:google-cloud-{service}-{version}-java',
   )
 
-java.common_templates()
+java.common_templates(excludes=[
+    '.gitignore',
+    '.github/CODEOWNERS',
```

Comment:
We really don't want to drift on the CODEOWNERs files unless we have a big need to